src/shape/arc.js

```diff
@@ -35,8 +39,12 @@ extend(ChartInternal.prototype, {
 		$$.radius = $$.radiusExpanded * 0.95;
 		$$.innerRadiusRatio = w ? ($$.radius - w) / $$.radius : 0.6;
 
+		const innerRadius = config.pie_gap ?
+			config.pie_gap : config.pie_innerRadius ?
```

Comment:
Using `pie_gap` value directly, will show inner circle when the gap value is up to 5.
- ex.) giving gap to 10, will display inner circle on orange colored area.
  ![image](https://user-images.githubusercontent.com/2178435/36654288-dc531562-1afe-11e8-99b0-60ff51728367.png)

Calculate with the `$$.innerRadiusRatio` as, 
```js
config.pie_gap * ($$.innerRadiusRatio + 0.1) : config.pie_innerRadius ?
```
will be rendered like

![image](https://user-images.githubusercontent.com/2178435/36654338-456d82e4-1aff-11e8-840f-7ca3a6f8987a.png)

Reply:
Yes looks like a good solution.
Thanks